docs/setup.md

```diff
@@ -2,8 +2,9 @@
 layout: page
 title: Using Babel
 description: How to use Babel with your tool of choice
-permalink: /docs/using-babel/
+permalink: /docs/setup/
 redirect_from:
+ - /docs/using-babel/
  - /plugins.html
  - /docs/setup/
```

Comment:
You created an infinite redirect loop here. `/docs/setup/ => /docs/setup/` just remove this line ^